presidio-anonymizer/presidio_anonymizer/anonymizers/hash.py

```diff
@@ -1,11 +1,12 @@
 """Hashes the PII text entity."""
+from presidio_anonymizer.anonymizers import Anonymizer
 
 
-class Hash(object):
+class Hash(Anonymizer):
```

Comment:
@shiranr this was not intentional right?

Reply:
Nope, an honest mistake. :)